src/Microsoft.SqlTools.Credentials/Program.cs

```diff
@@ -0,0 +1,47 @@
+//
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using Microsoft.SqlTools.Credentials.Hosting;
+using Microsoft.SqlTools.Credentials.Utility;
+using Microsoft.SqlTools.ServiceLayer.SqlContext;
+using Microsoft.SqlTools.ServiceLayer.Utility;
+
+namespace Microsoft.SqlTools.ServiceLayer
+{
+    /// <summary>
+    /// Main application class for Credentials Service Host executable
+    /// </summary>
+    internal class Program
+    {
+        /// <summary>
+        /// Main entry point into the Credentials Service Host
+        /// </summary>
+        internal static void Main(string[] args)
```

Comment:
Is the goal of this to create a new service host that will implement its own JSON RPC just for servicing credentials requests? Or is the goal to spin the credentials service out to its own library that the tools service (and potentially other tools services) would consume? I think if we can go the latter route we'll reduce overhead tremendously and drastically reduce the amount of duplicated code.